includes/data-stores/class-wc-admin-reports-customers-data-store.php

```diff
@@ -250,7 +260,7 @@ protected function get_sql_query_params( $query_args ) {
 
 		// Allow a list of customer IDs to be specified.
 		if ( ! empty( $query_args['customers'] ) ) {
-			$included_customers = implode( ',', $query_args['customers'] );
+			$included_customers = implode( ',', esc_sql( $query_args['customers'] ) );
```

Comment:
`esc_sql()` is meant for string values that will be in quotes in the SQL statement - we should probably `intval()` each customer ID instead.